automl/google/cloud/automl_v1beta1/helper/tables.py

```
@@ -1002,21 +949,43 @@ def create_model(self, model_display_name, dataset=None,
                 dataset_display_name=dataset_display_name,
                 project=project,
                 region=region)
-
+        tables_model_metadata = {
+                'train_budget_milli_node_hours': train_budget_milli_node_hours
+        }
         dataset_id = dataset_name.rsplit('/', 1)[-1]
+        columns = [s for s in self.list_column_specs(dataset=dataset, dataset_name = dataset_name, dataset_display_name=dataset_display_name)]
+
+        final_columns = []
+        if input_feature_column_specs_included:
+            for a in columns:
+                if a.display_name in input_feature_column_specs_included:
+                    final_columns.append(a)
```

Since we know the list of columns, I would raise a `ValueError` here if a user supplied an invalid column name. I think that might save some headaches for the user down the road.

---

will add